grouper/constants.py

```diff
@@ -19,6 +19,9 @@
 PERMISSION_GRANT = "grouper.permission.grant"
 PERMISSION_CREATE = "grouper.permission.create"
 PERMISSION_AUDITOR = "grouper.permission.auditor"
+PERMISSION_ADMIN = "grouper.admin.permissions"
+USER_ADMIN = "grouper.admin.users"
+GROUP_ADMIN = "grouper.admin.groups"
```

Comment:
I ported the semantics over directly from the old capabilities model. We can make things more fine-grained as we desire.

I put these under `grouper.admin.*`  to make it obvious that they're dangerous. But happy to swap the last two elements in the hierarchy for each.